app/forms/state_file/nj_eligibility_health_insurance_form.rb

```diff
@@ -1,11 +1,22 @@
 module StateFile
   class NjEligibilityHealthInsuranceForm < QuestionsForm
     set_attributes_for :intake, :eligibility_all_members_health_insurance
+    attr_accessor :dependents_attributes
+    delegate :dependents, to: :intake
 
     validates :eligibility_all_members_health_insurance, presence: true
 
     def save
       @intake.update(attributes_for(:intake))
+      if @intake.eligibility_all_members_health_insurance_no?
```

Comment:
really appreciate the attention to detail here. these nested forms have introduced a fair number of data inconsistency issues for us in the past, so i'm glad that you were able to follow the examples in the codebase of how we've overwritten params to make sure that the sub-fields are made consistent with the "top-level" answers. 🙌